spec/services/checksum_validator_spec.rb

```diff
@@ -203,24 +203,60 @@
         expect(results).to receive(:add_result).with(
           AuditResults::INVALID_MANIFEST, manifest_file_path: 'spec/fixtures/checksum_root01/moab_storage_trunk/xx/444/xx/4444/xx444xx4444/v0001/manifests/signatureCatalog.xml'
         )
-        cv.validate_signature_catalog
+        cv.send(:validate_signature_catalog_listing)
       end
     end
   end
 
-  context '#validate_checksum' do
+  context '#flag_unexpected_data_content_files' do
     let(:druid) { 'bj102hs9687' }
-    let(:cv) { described_class.new(druid, 'fixture_sr1') }
+    let(:cv) { described_class.new(druid, endpoint_name) }
 
-    it 'returns a positive result for a druid that passes validation' do
-      cv.validate_checksum
-      expect(cv.checksum_results.result_array.first).to have_key(:moab_checksum_valid)
+    it 'calls validate_against_signature_catalog on each of the data_content_files' do
+      files = ['spec/fixtures/storage_root01/moab_storage_trunk/bj/102/hs/9687/bj102hs9687/v0001/data/content/eric-smith-dissertation-augmented.pdf',
```

Comment:
pro tip:  put `'spec/fixtures/storage_root01/moab_storage_trunk/bj/102/hs/9687/bj102hs9687/v0001/data/` into a variable (maybe in a let statement) and then make it easy to read files:

```ruby
files = [
    "#{object_version_data_dir}metadata/asdfasf.xml",
    "#{object_version_data_dir}content/ugh.xml",
    "#{object_version_data_dir}content/subdir/foo.xml"
]
```
Then it's really easy to see what you are actually testing.